src/key_schedule.cpp

```diff
@@ -275,8 +275,8 @@ KeyScheduleEpoch::next(const bytes& commit_secret,
                        const bytes& context,
                        LeafCount size) const
 {
-  auto joiner_secret = suite.get().hpke.kdf.extract(init_secret, commit_secret);
-  return KeyScheduleEpoch(suite, joiner_secret, psk_secret, context, size);
+  auto secret = suite.get().hpke.kdf.extract(init_secret, commit_secret);
+  return KeyScheduleEpoch(suite, secret, psk_secret, context, size);
```

Comment:
`joiner_secret` masks `KeyScheduleEpoch::joiner_secret` and causes warning during compilation about masking class member.

Reply:
@bifurcation , are you ok to rename `joiner_secret` to something different to avoid warning message about masking ?
Richard, I don't have macos machine and I hardly understand this error :
`/Users/runner/work/mlspp/mlspp/lib/hpke/src/group.cpp:219:60: error: consider replacing 'long' with 'int64' [google-runtime-int,-warnings-as-errors]
        o2i_ECPublicKey(&eckey_ptr, &data_ptr, static_cast<long>(enc.size()))) {`

Reply:
On `joiner_secret` - Good point.  How about `temp_joiner_secret`?

That error is from clang-tidy.  I would try replacing `<long>` with `<int64>`.  If that makes Windows unhappy, then just add a `// NOLINTNEXTLINE(google-runtime-int)` to silence the error.

Reply:
@bifurcation , thanks for hint. I think PR is ready for merge now.